manual/Makefile

```diff
@@ -1,9 +1,17 @@
 # Makefile for the reference and user manuals
 TEXDIR=tex_files
 VPATH=$(TEXDIR)
-.PHONY:all
+.PHONY:all clean sphinx
 
-all: user_manual.pdf \
+all: sphinx
+
+sphinx:
+	sphinx-build -b latex sphinx/ build/
+	cd build && latexmk -pdf BOUT
+	test -e BOUT.pdf || ln -s build/BOUT.pdf .
+	@echo "Documentation is available in BOUT.pdf"
+
+old: user_manual.pdf \
```

Comment:
I changed `make.config` to forward to the instructions in `manual/Makefile`.
Further, the README should be containing the proper instructions.